app/views/manifests/index.html.erb

```diff
@@ -1,6 +1,10 @@
 <div class="usa-width-two-thirds">
 
-  <h3>Public manifests</h3>
+  <h3>Manifests</h3>
+
+  <div class="stats">
+    Currently viewing <%= @stats[:from] %>-<%= @stats[:to] %> of <%= number_with_delimiter @stats[:total] %> results.
```

Comment:
not sure what the standard # of records for pagination is or how it is being set -- would it be possible to write a feature level test for this pagination? Ideally, we could set records per page to "1" and test that when we create 2 records that match a seacrch, only one of them is on the page.